common/tools/dev-tool/src/commands/run/testVitest.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import concurrently from "concurrently";
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+import { runTestsWithProxyTool } from "../../util/testUtils";
+import { createPrinter } from "../../util/printer";
+
+export const commandInfo = makeCommandInfo(
+  "test:vitest",
+  "runs tests using vitest with the default and the provided options; starts the proxy-tool in record and playback modes",
+  {
+    "no-test-proxy": {
+      shortName: "ntp",
+      kind: "boolean",
+      default: false,
+      description: "whether to disable launching test-proxy",
+    },
+    "browser": {
+      shortName: "br",
+      kind: "boolean",
+      default: false,
+      description: "whether to use browser to run tests",
+    },
+  },
+);
+
+export default leafCommand(commandInfo, async (options) => {
+  const args = options["browser"] ? "-c vitest.browser.config.mts": "";
```

Comment:
should we also have this branch handle the playwright install? or is that getting too fancy for the first PR?